src/core/DataFile.cpp

```diff
@@ -1684,6 +1685,50 @@ void DataFile::upgrade_noteTypes()
 	}
 }
 
+void DataFile::upgrade_fixCMTDelays()
+{
+	QMap<QString, QString> nameMap;
+	nameMap["delay_0,01s"] = "delay_0.01s";
+	nameMap["delay_0,1s"] = "delay_0.1s";
+	nameMap["fbdelay_0,01s"] = "fbdelay_0.01s";
+	nameMap["fbdelay_0,1s"] = "fbdelay_0.1s";
```

Comment:
This can probably be a static map?
```suggestion
	static const QMap<QString, QString> nameMap{
		{"delay_0,01s", "delay_0.01s"},
		{"delay_0,1s", "delay_0.1s"},
		{"fbdelay_0,01s", "fbdelay_0.01s"},
		{"fbdelay_0,1s", "fbdelay_0.1s"}
	};
```

Reply:
Adjusted with commit e1899d82755.